src/board/mod.rs

```diff
@@ -243,19 +257,90 @@ impl Board {
             let color = self.next_player();
             self.vacant
                 .iter()
-                .map(|coord| Play(color.clone(), coord.col, coord.row))
+                .map(|coord| Play(color, coord.col, coord.row))
                 .filter(|m| self.is_legal(*m).is_ok())
                 .collect()
         }
     }
-
```

Comment:
OK, so let's leave it there for now. I'll have a look at it in my branch as I work on having two kinds of playouts (one with the eye check and one without it) anyway.